tests/components/hue/test_light_v1.py

```diff
@@ -178,6 +179,8 @@ async def setup_bridge(hass, mock_bridge_v1):
     """Load the Hue light platform with the provided bridge."""
     hass.config.components.add(hue.DOMAIN)
     config_entry = create_config_entry()
+    config_entry.add_to_hass(hass)
+    config_entry.state = ConfigEntryState.LOADED
```

Comment:
This ensures that the corresponding config entry is unloaded at the end of the tests.